app/services/paper_converters/supporting_information_file_proxy.rb

```diff
@@ -52,7 +52,7 @@ def self.from_snapshot(snapshot)
       token = snapshot.get_property("url").split('/').last
       new(
         filename: snapshot.get_property("file"),
-        resource_token: ResourceToken.find_by!(token: token),
+        resource_token: ResourceToken.find_by(token: token),
```

Comment:
should this fail hard?

Reply:
Probably, but there is some data that fails.